File: app-core/src/main/java/kr/flab/wiki/core/login/persistence/UserEntity.java
package kr.flab.wiki.core.login.persistence;

public class UserEntity implements User {

    private String email;
    private String password;

    public UserEntity(String email, String password){
        this.email = "";
        this.password = "";
    }

    @Override
    public String getEmail() {
        return this.email;
    }

    @Override
    public String getPassword() {
        return this.password;
    }
}

File: app-core/src/main/kotlin/kr/flab/wiki/core/login/persistence/UserEntity.kt
package kr.flab.wiki.core.login.persistence

import kr.flab.wiki.core.login.LoginUtils
import kr.flab.wiki.core.login.exception.UserValidationException

class UserEntity : User {
    override var email : String = ""
//        set(value) {
//            if(value.isEmpty()){
//                throw UserValidationException("Email is Empty!")
//            }
//            if (!LoginUtils.isValidEmail(value)) {
//                throw UserValidationException("Not Email Type!")
//            }
//            field = value
//        }
    override var password : String = ""
//        set(value) {
//            if (value.isEmpty()) {
//                throw UserValidationException("Password is Empty!")
//            }
//            field = value
//        }
}
